docs/process/integration-with-subscription-platform.md

```diff
@@ -4,28 +4,108 @@ title: Integration with Subscription Platform
 
-## For Product Managers
+### FxA Event Broker
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chick
+Service that distributes [relying party events] to FxA integrations that have a registered [webhook] URL. These events are only sent to integrations that a FxA user has logged into.
 
-## For Engineers
+## Pre-Development
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chick
+Before getting started, you should have had a RRA-style meeting with the Firefox Accounts team. During or shortly after this meeting, an initial subscription capability string should have been agreed upon. If the product/plan did not exist yet, one will be created for the FxA stage environment.
 
-## For Designers
+You will need up to three defined strings to develop a subscription platform integration:
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chick
+- Subscription Capability string
+- Product id (If building a lead page)
+- Plan id (If building a lead page)
+
+If your integration includes an application service, it will need a [webhook] handler to recieve [relying party events] from the FxA Event Broker.
+
+## Development
+
+Integration with FxA is the primary development task to complete for a working subscription integration. The additional Subscription Capability string will be included as the `subscriptions` field in the response from the FxA Profile Server.
+
+### Lead Pages
+
+If your integration includes lead pages to start a subscription flow, you will need to include the product and plan id's in the subscription URL.
```

Comment:
@lmorchard I need some help here on how we have this setup, where do the query params go, what URL's to use, anything else needed to setup the subscription flow for a new RP, etc.

Reply:
Basic product payment page URL template is this:
```
https://accounts.firefox.com/subscriptions/products/{productId}?plan={planId}
```
And `accounts.firefox.com` can be replaced with another env host (e.g. `accounts.stage.mozaws.net` for stage, `latest.dev.lcip.org` for dev). But, of course, valid values for `productId` and `planId` will vary depending on what's in the Stripe account associated with the environment.

Are there other URLs you're looking for here?

Reply:
Oh yeah, and the subscriptions management link should be:
```
https://accounts.firefox.com/subscriptions/
```
Trying to think if there are any other URLs a lead page might need, but those are probably the only two?

Reply:
Nope, those are the ones!

Reply:
Not sure the subscriptions link is actually need, I'll leave it off for now.